api/SourceBuffer.json

```diff
@@ -477,10 +518,10 @@
               "version_added": false
             },
             "samsunginternet_android": {
-              "version_added": "3.0"
+              "version_added": false
```

Comment:
The data that was here was just wrong, probably copy-pasted at one point in time.